scripts/box

```diff
@@ -433,7 +435,7 @@ function _chpasswd() {
     fi
     if [[ -f /install/.qbittorrent.lock ]]; then
         . /etc/swizzin/sources/functions/qbittorrent
-        qbittorrent_chpasswd ${user} ${pass}
+        qbittorrent_chpasswd "${user}" "${pass}"
```

Comment:
This will work I'm assuming?

Reply:
tested anbd verified, just needs to aactually restart the service again afterwards though